web/src/stream_edit.js

```diff
@@ -595,20 +595,21 @@ export function initialize() {
         }
 
         const stream = sub_store.get(stream_id);
-        const stream_privacy_symbol_html = render_stream_privacy({
+        const stream_name = stream_data.maybe_get_stream_name(stream_id);
+
+        const stream_name_with_privacy_symbol_html = render_inline_decorated_stream_name({
+            stream_name,
             invite_only: stream.invite_only,
             is_web_public: stream.is_web_public,
         });
```

Comment:
Would it be cleaner to have `render_inline_decorated_stream_name` accept a `stream` object as its parameter, rather than a few fields from it? I guess I don't know what `stream_data.maybe_get_stream_name` is doing here.

Reply:
Udpated the PR to pass stream object to the template. Also, we do not need `stream_data.maybe_get_stream_name` here so we can pass stream obejct directly.